cypress-tests/cypress/e2e/PaymentUtils/Fiuu.js

```diff
@@ -360,6 +360,28 @@ export const connectorDetails = {
         },
       },
     },
+    MITAutoCapture: {
+      Request: {},
+      Response: {
+        status: 200,
+        body: {
+          status: "failed",
+          error_code: "The currency not allow for the RecordType",
+          error_message: "The currency not allow for the RecordType",
+        },
```

Comment:
this body could have been re-used.

Reply:
Just two places we use this

Reply:
It's just the error message right , should we make the changes ?

Reply:
if it can help improve code readibility, re-usabulity, i feel, we should go ahead and do it